app/src/main/java/com/jshvarts/todoapp/MainActivity.kt

```diff
@@ -1,52 +1,22 @@
 package com.jshvarts.todoapp
 
-import android.annotation.SuppressLint
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.annotation.StringRes
```

Comment:
moved screen list related code to screenlist.kt and screen detail code to screendetail.kt